build-support/bin/_release_helper.py

```diff
@@ -70,7 +70,7 @@
 
 _expected_owners = {"benjyw", "John.Sirois", "stuhood"}
 
-_expected_maintainers = {"EricArellano", "illicitonion", "wisechengyi"}
+_expected_maintainers = {"EricArellano", "illicitonion", "wisechengyi", "kaos"}
```

Comment:
what if the expected maintainers check was done as a subset check rather than an exact match? as this feels rather brittle, as soon as we add another (Dan M) releaser it's going to break again.
I guess this is a sanity check to ensure we're working with the proper pypi credentials to avoid mistakes, so a subset check ought to serve just as well in that regard.